ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/ScanConfigsHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+    Flex,
+    PageSection,
+    Title,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+
+type ScanConfigsHeaderProps = {
+    description: string;
+    actions: React.ReactNode;
+};
+
+function ScanConfigsHeader({ description, actions }: ScanConfigsHeaderProps) {
+    return (
+        <>
+            <PageTitle title="Scan Schedules" />
+            <PageSection variant="light">
+                <Title headingLevel="h1">Scan schedules</Title>
+            </PageSection>
+            <PageSection variant="light" className="pf-u-py-0">
```

Comment:
Aha, when I applied similar examples from elsewhere, Mansur asked me to remove the double Xl vertical spacing (which I notice in the corresponding picture).

Here is a recent example from Workload CVEs for your consideration:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx#L89-L112